features/Functional-parts.md

```diff
@@ -27,3 +27,4 @@
 | PSE  | Property Sets                    | @PSE        |                                                  |
 | PJS  | Project Definition               | @PJS        |                                                  |
 | BLT  | Built Elements                   | @BLT        |                                                  |
+| SPA  | Spaces Infomation                | @SPA        |                                                  |
```

Comment:
```suggestion
| SPA  | Space Infomation                | @SPA        |                                                  |
```

Think this is better english.

Reply:
This file doesn't exist anymore. @Ghesselink ?

Reply:
No, indeed. This is the disadvantage of having too many open pull requests. The one that merged the changes that you mention are not reflected in the current pull request. 
This caused some merged conflicts; just resolved them; https://github.com/buildingSMART/ifc-gherkin-rules/pull/246/commits/a1b1c3da2336742031d50dde7e65d849e85cc1c2